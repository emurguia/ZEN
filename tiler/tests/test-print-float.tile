// tests that floats can be printed

init {
	float x;
	tile(3,3);
	x = 3.3;
	fprint(x);
	close();
}