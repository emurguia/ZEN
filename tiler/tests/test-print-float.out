3.300000
The game window was successfully launched ...
The game window has been exited. Program exited.
Game over.